sentry_sdk/integrations/pymongo.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+
+from sentry_sdk import Hub
+from sentry_sdk.integrations import DidNotEnable, Integration
+from sentry_sdk.tracing import Span
+from sentry_sdk.utils import capture_internal_exceptions
+
+from sentry_sdk._types import MYPY
+
+try:
+    from pymongo import monitoring
+except ImportError:
+    raise DidNotEnable("Pymongo not installed")
```

Comment:
Hi,
May be would be nice also to explicitly check the pymongo version and give the more specific alert as in SqlalchemyIntegration, for example, because pymongo could be installed but has the wrong version.

Reply:
I can add it although barely any integration does it - even new ones, like FastAPI just check for the import itself.

Reply:
I am not sure that FastAPI has any restrictions in its version to be able to use the integration. Anyway, It was just a proposal.

Reply:
It does - otherwise there would be no minimal version specified in `setup.py`. For example see https://github.com/getsentry/sentry-python/pull/1349 - FastAPI `0.74.1` had dedicated changes to enable transaction name retrieval. And I believe there were further changes in it to improve integration.

Reply:
Hey! I guess the import check as you implemented is totally fine.